apps/wallet-mobile/src/features/Swap/useCases/ConfirmTxScreen/ConfirmTxScreen.tsx

```diff
@@ -68,33 +86,36 @@ export const ConfirmTxScreen = () => {
               authWithOs()
               return
             }
-            openBottomSheet()
+            openConfirmTxBottomSheet()
           }}
         />
       </Actions>
 
       <BottomSheet
         height={wallet.isHW ? 430 : 350}
-        ref={bottomSheetRef}
+        ref={confirmTxBottomSheetRef}
         title={wallet.isHW ? strings.chooseConnectionMethod : strings.signTransaction}
-        isExtendable={false}
       >
         <View style={styles.modalContent}>
           <ConfirmTx
             wallet={wallet}
             unsignedTx={unsignedTx}
             datum={{data: createOrder.datum}}
             onSuccess={() => {
-              closeBottomSheet()
+              closeConfirmTxBottomSheet()
               navigate.submittedTx()
```

Comment:
is there any animation on the bottomSheet? if there is it needs to be wrapped within `InteractionManager.runAfterInteractions`

Reply:
default react native animation modal